From 9ea6fe2e4145d2fb7bda115514824f47935e6236 Mon Sep 17 00:00:00 2001
From: crazy <crazy@frugalware.org>
Date: Fri, 9 Oct 2015 06:03:13 +0200
Subject: [PATCH 2/2] makepkg * well we really want to know what C/XX/LD flags
 are used * to build the package. I may be some devels are using wrong flags..

---
 scripts/makepkg | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/scripts/makepkg b/scripts/makepkg
index 8fff977..71859a8 100755
--- a/scripts/makepkg
+++ b/scripts/makepkg
@@ -1877,6 +1877,11 @@ elif [ "$RMDEPS" = "1" -a "$DEP_SUDO" = "1" ]; then
 	fi
 fi
 
+msg "This package was build with:"
+msg2 "HOST_CFLAGS:    $CFLAGS "
+msg2 "HOST_CXXFLAGS:  $CXXFLAGS "
+msg2 "HOST_LDFLAGS:   $LDFLAGS "
+
 msg "Finished making: $pkgname  (`date`)"
 
 install_pkg
-- 
2.5.0

